css/style.css

```diff
@@ -14,6 +14,10 @@ header h1+h1 {
     display: none;  /* disable link and time on very small devices, will be enabled on bigger screens */
 }
 
+header h1:first-child span {
```

Comment:
For this one it makes sense, but for the others I am not sure which others you mean. For e.g. `section#daytoggler button:first-child / button:last-child` it makes perfect sense to go without fixed ids.

I can imagine that for header h1+h1 it could be better to have an id, but that's a little bit unrelated.

I will fix the marked line and add an id for that element.

Reply:
Thanks :). It was more like a general comment. I saw you using those selectors more and more and I think they are a little fragile, whereas ids seem more appropriate to me.